File: 240306/Next Level/next-level.py
class Game:
  def __init__(self, id, level):
    self.id =id
    self.level = level

player1 = Game('codetree', 10)
print(f'user {player1.id} lv {player1.level}')

a, b = input().split()
player2 = Game(a,b)

print(f'user {player2.id} lv {player2.level}')

File: 240423/Next Level/next-level.js
const fs = require('fs')
const input = fs.readFileSync(0).toString().trim().split(' ')
const a = input[0]
const b = Number(input[1])

class Game {
    constructor(id = 'codetree', level = 10) {
        this.id = id;
        this.level =level;
    }
}

const account1 = new Game()
console.log(`user ${account1.id} lv ${account1.level}`)
const account2 = new Game(a, b)
console.log(`user ${account2.id} lv ${account2.level}`)